src/Network/Network/help/Set-AzVirtualNetworkSubnetConfig.md

```diff
@@ -81,6 +81,20 @@ This example creates a resource group with one virtual network containing just o
     security group. The Set-AzVirtualNetwork cmdlet is then called to write the modified 
     state back to the service.
 
+### 3: Attach a Nat Gateway to a subnet
```

Comment:
This help file is for Set-AzVirtualNetworkSubnetConfig and there is no `Set-` call in the example, please modify.